### PR TITLE
Added Tier

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 601b96ee9a8b9257bdfcb70ae31cb52180fb7039:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/services/test_service_catalogs.py [hunk 1]
```diff
@@ -16,7 +16,8 @@
 pytestmark = [
     pytest.mark.meta(server_roles="+automate"),
     pytest.mark.usefixtures('logged_in', 'vm_name', 'catalog_item', 'uses_infra_providers'),
-    pytest.mark.long_running
+    pytest.mark.long_running,
+    pytest.mark.tier(1)
```
Comment (mfalesni) [inline cfme/tests/services/test_service_catalogs.py]: You sure about tier 1? I thought we don't have tier 1 tests in cfme_tests.

Comment (sshveta) [inline cfme/tests/services/test_service_catalogs.py]: By mistake . I meant to add tier 2 . Getting forgetful these days :)

**Comment by dajoRH** [comment]
Lint report for commit fb554f1255ac27a0a7ea871d53f1bbf64a872cb0:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/services/test_service_catalogs.py [hunk 2]
```diff
@@ -16,7 +16,8 @@
 pytestmark = [
     pytest.mark.meta(server_roles="+automate"),
     pytest.mark.usefixtures('logged_in', 'vm_name', 'catalog_item', 'uses_infra_providers'),
-    pytest.mark.long_running
+    pytest.mark.long_running,
+    pytest.mark.tier(2)
```
Comment (psav) [inline cfme/tests/services/test_service_catalogs.py]: I think this should be tier(3)

Comment (sshveta) [inline cfme/tests/services/test_service_catalogs.py]: Tier 2 : Integration level functional tests . service catalogs for all providers will come under this . No ?? Just trying to understand . If not then what will come under Tier 2 ?

Comment (psav) [inline cfme/tests/services/test_service_catalogs.py]: Tier 2 are appliance level smoke tests really, some of these tests look like actually they could be tier 2, but I don't think all of them.

**Comment by dajoRH** [comment]
Lint report for commit 265c40225670f02972d88b153dc8747f21e2adcf:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**